Ginger/GingerCore/ALM/OctaneCore.cs

```diff
@@ -1,20 +1,4 @@
-#region License
-/*
-Copyright © 2014-2020 European Support Limited
 
```

Comment:
Copy right license is required. Please add it back

Reply:
Added license again